### DISCUSSION TITLE
3D Model is cut off by a div on site

### DISCUSSION THREAD

**Discussion by idrisbrewster** [google/model-viewer#2944] [Q&A]
I am looking to make the scale of my 3d model bigger so that it takes up a larger amount of the page, but there is some sort of div/box that is cuts it off when I make the scale bigger. Is there anyway to increase the size of that container so that I can make my model bigger without cutting any of it off.

Here is the site: https://moversandshakersnyc.webflow.io/about

**Comment by elalish** [comment]
It appears you have not set the CSS `width` or `height` on your MV element, so it's just using the default 300x150 pixels.

**Reply by idrisbrewster** [reply]
Not sure what I am doing wrong, but I specified the MV width and height and its still giving me the same issues

`<model-viewer id="toussaint"alt="Toussaint" src="https://uploads-ssl.webflow.com/615b5aa6b965137581a5be4a/61771ecc1b696b081a62e355_toussaintWeb.glb.txt" scale="0.001 0.001 0.001" width="500px" height="750px" camera-controls></model-viewer>`

**Reply by idrisbrewster** [reply]
I've also brought up the developer tools on the page and added width and height and changed around the numbers from other lines of code and nothing has been able to increase the size of that box

**Reply by elalish** [reply]
You need to use CSS; width and height are not attributes. It's exactly like setting the size of a `<div>`.

**Reply by idrisbrewster** [reply]
so something like this:

toussaint {
  width: 500px;
  height: 750px;
}

**Reply by idrisbrewster** [reply]
I am able to change the size of that box through developer tools. The CSS properties that I am changing are as follows:

`:host {
    display: block;
    position: relative;
    contain: strict;
    width: 300px;
    height: 150px;
}`

However when I try to change the CSS of the property using its id, it doesn't change. I have tried using ":host", "toussaint" and "model-viewer#toussaint" as the selector, and none of those have worked. How can i implement these changes in my CSS?

**Reply by ghost** [reply]
try adding `model-viewer#toussaint` to your styles

example:
```
<style>
  model-viewer#toussaint {
    width: 500px;
    height: 750px;
  }
</style>
```
also this should work
```
#toussaint {
width: 500px;
height: 750px;
}
```

**Reply by idrisbrewster** [reply]
this worked! Thank you!!
